content/docs/sidebar.json

```diff
@@ -620,5 +620,94 @@
         "source": "docs.md"
       }
     ]
+  },
+  {
+    "slug": "gto",
+    "label": "GTO",
+    "source": "gto/index.md",
+    "children": [
+      {
+        "slug": "install",
+        "source": "install.md",
+        "label": "Installation"
+      },
+      {
+        "slug": "get-started",
+        "label": "Get Started",
+        "source": "get-started.md"
+      },
+      {
+        "slug": "user-guide",
+        "label": "User Guide",
+        "source": "user-guide.md"
+      },
+      {
+        "slug": "dvc",
+        "source": "dvc.md",
+        "label": "GTO with DVC"
+      },
+      {
+        "slug": "mlem",
+        "source": "mlem.md",
+        "label": "GTO with MLEM"
```

Comment:
I changed the structure.
1. Removed Use Cases for now, redirecting to MLEM's MR UC
2. Added top-level "GTO with..." sections
3. Collapsed all UG subpages into a single page

Suggesting to merge this, and then improve. WDYT?

Reply:
@jorgeorpinel, please approve this PR and merge) I'll create the next "improvement PR" then.

Reply:
Sorry @aguschin, only getting back to this now. Unfortunately many of these docs are totally new to me still (even higher level ones like User Guide, Why GTO) so I left another round of comments on everything other than the Get Started. It's still a pretty huge PR, not easy to move fwd. Maybe just the GTO docs index and Get Started can be merged first? (Would need to get extracted to another PR).

For now I'll do a final pass on the GS ⏳

Reply:
> p.s. of course if you prefer to get this approved by anyone and merged as-is, that's your choice. I'm not trying to block it. I would recommend my divide-and-conquer strategy more though, so we can really boil all this down to what we absolutely need (less docs = easier maintenance).

Reply:
Thanks. Let's merge everything that is ready now and then improve.